modules/smtp/hm-smtp.php

```diff
@@ -60,6 +60,9 @@ class Hm_SMTP {
     private $config;
     private $server;
     private $starttls;
+    private $supports_tls;
```

Comment:
These are not really related to sieve corrections, just some deprecations that i've been getting while correcting.
<img width="1116" alt="Screenshot 2024-08-27 at 05 20 41" src="https://github.com/user-attachments/assets/573825c2-e931-46b2-8478-11c821be6c69">